service_tools/pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[project]
+version = "1.0.0"
+name = "shotgrid-service-tools"
+requires-python = ">=3.7"
```

Comment:
This will not work as `services` require Python `>=3.10,<4.0`

Reply:
I don't think it's a big deal, but ok, changed.